js/storage.ts

```diff
@@ -1,10 +1,12 @@
-import firebase from "firebase/app";
-import "firebase/database";
-import { v4 as uuidv4 } from "uuid";
+import { FirebaseApp, initializeApp } from "firebase/app";
+import { set, ref, onValue, getDatabase } from "firebase/database";
```

Comment:
I'm glad to see that you took the plunge to the new v9 API without too much difficulty. I've been holding off on upgrading on several projects that use firebase more extensively. From this example, however, it looks like the changes are pretty mechanical.

Reply:
They are, but almost every line will be affected. 😉 
Some functions are gone too and there's no direct equivalent (like `once` => `onValue` + custom options). But the new API definitely looks nicer, less magic (especially with the app config), more explicit.

Reply:
>The splash screen in TE is broken without that rule

That also suggests that it's a new default optimization in v6 given that you weren't experiencing it before. Given that, I'm surprised we haven't seen more breakage. 🤔